src/arkode/arkode_io.c

```diff
@@ -1995,6 +1953,68 @@ int ARKodeSetMaxConvFails(void* arkode_mem, int maxncf)
   return (ARK_SUCCESS);
 }
 
+/*---------------------------------------------------------------
+  ARKodeSetAccumulatedErrorType:
+
+  This routine sets the accumulated temporal error estimation
+  strategy.
+  ---------------------------------------------------------------*/
+int ARKodeSetAccumulatedErrorType(void* arkode_mem, ARKAccumError accum_type)
+{
+  ARKodeMem ark_mem;
```

Comment:
I don't think it reduces anything.  Yes, it would remove two lines that update `AccumErrorStep` and `AccumError`, but it would add redundant tests for `(arkode_mem == NULL)` and `(!ark_mem->step_supports_adaptive)`.